internal/builders/go/pkg/build.go

```diff
@@ -52,6 +52,8 @@ var allowedEnvVariablePrefix = map[string]bool{
 	"GO": true, "CGO_": true,
 }
 
+var Output = os.Stdout
```

Comment:
I'm thinking this should be a field on `GoBuild` rather than a package level global. That way all GoBuild instances aren't bound to using a single writer. It should also allow tests to run in parallel.